api/tests/mocha/services/users.spec.js

```diff
@@ -75,6 +75,12 @@ describe('Users service', () => {
       chai.expect(settings.fullname).to.equal('John');
     });
 
+    it('does not reassign language', () => {
```

Comment:
Please add some tests here that make sure we maintain backwards compatibility: 
1. Online user requests with just `language` field shouldn't reject with 400.
2. Offline user requests that include `language` field shouldn't reject with 401.